docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,170 @@
+
+You, the customer, own and are responsible for your test content. We trust that customers will make decisions about the appropriateness of data to use in testing and will avoid use of PII, PHI, or other types of protected information. Cypress makes commercially reasonable efforts to protect confidentiality of information that is provided to Cypress Cloud.
+
+## Cypress Cloud data controls
+
+We understand that controlling what data is sent to Cypress Cloud is important to our users. Cypress provides a number of ways to control what data is sent to Cypress Cloud.
+
+If you'd like more granular control over what data is sent to Cypress Cloud, please [contact us](https://www.cypress.io/contact/) to discuss your needs.
+
+### Project access controls
+
+Cypress Cloud offers controls around the visibility of a project. You can set a project to be public or private. [See project access control settings.](/guides/cloud/account-management/projects#Public-vs-Private)
+
+Setting a project to private means that only the users explicitly invited to the organization will have visibility into the project, its runs, and any data captured by Cypress.
+
+### Test Replay controls
+
+#### Disable Test Replay
+
+You can disable capturing data in Test Replay per project by following [this guide](/guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay). This means Test Replay will not be available for viewing or debugging in the project. This will prevent Cypress Cloud from capturing the following data:
+
+- The rendered DOM and CSS styles for the application under test
+- Cypress commands and events represented in the Command Log
+- Network traffic within your application under test
+- Browser console logs
+
+### Screenshot controls
+
+#### Blackout elements in screenshots
+
+Cypress offers an option to black out (mask) an array of elements from screenshots. [See screenshot API docs.](/api/commands/screenshot#Arguments) This allows you to black out specific elements on screenshots so that they will not be visible in the locally saved screenshots or to any users that have access to the run in Cypress Cloud.
+
+Add a control to your [support file](/guides/references/configuration#Testing-Type-Specific-Options) to set the default blackout options for all screenshots.
+
+```js
+before(() => {
+  Cypress.Screenshot.defaults({
+    // blackout all elements with 'mask-cy' attribute
+    blackout: ['mask-cy'],
+  })
+})
+```
+
+Add a control for a specific screenshot to blackout specific elements.
+
+```js
+it('admin page loads', () => {
+  cy.visit('/admin')
+  cy.screenshot({
+    blackout: ['.user-name'],
+  })
+})
+```
+
+#### Remove Cypress Command Log from screenshots
+
+Cypress offers an option to hide the Cypress Command Log from screenshots so that no information from that area will be captured in screenshots. [See `cy.screenshot` docs](/api/commands/screenshot#Arguments).
```

Comment:
some of these paragraphs feel like they end a bit abruptly. Not sure if this is the best suggestion, but it feels at least a bit more consistent.
```suggestion
Cypress offers an option to hide the Cypress Command Log from screenshots so that no information from that area will be captured in screenshots. See the [`cy.screenshot()`](/api/commands/screenshot#Arguments) command for more information.
```